pages/packages/private_storage.md

```diff
@@ -0,0 +1,42 @@
+# Link private storage
+
+This page provides details on how to link your private Amazon Web Services (AWS) Simple Storage Service (S3) storage to Buildkite Packages within your Buildkite organization. This process can only be conducted by [Buildkite organization administrators](/docs/packages/permissions#manage-teams-and-permissions-organization-level-permissions).
+
+By default, Buildkite Packages provides its own storage to house any packages, container images and modules stored in registries. You can also link your own private AWS S3 bucket to Buildkite Packages, which allows you to:
+
+- Manage Buildkite registry packages, container images and modules stored within your private AWS S3 bucket (that is, your _private storage_). Private storage located closer to your geographical location may provide faster registry access.
```

Comment:
I feel like a key benefit of private storage isn't quite hit here — that using an S3 bucket means avoiding network transmission costs.

Reply:
Thanks I'll tweak this accordingly.